testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/forms/RegisterTest.java

```diff
@@ -187,11 +189,11 @@ public void registerPasswordPolicy() {
         /*keycloakRule.configure(new KeycloakRule.KeycloakSetup() {
             @Override
             public void config(RealmManager manager, RealmModel adminstrationRealm, RealmModel appRealm) {
-                appRealm.setPasswordPolicy(new PasswordPolicy("length"));
+                appRealm.setPasswordPolicy(new PasswordPolicy(LengthPasswordPolicyProviderFactory.ID));
```

Comment:
Please remove this. Tests are by intent using a string and not a constant here. When using a constant one example would be that someone by mistake renames the ID of the provider resulting in the test passing, but existing password policy strings would no longer work.

Reply:
done